db/migrate/20181126215611_add_authentications.rb

```diff
@@ -0,0 +1,17 @@
+class AddAuthentications < ActiveRecord::Migration[5.1]
+  def change
+    create_table :authentications, :id => :bigserial do |t|
+      t.string     :name
+      t.string     :authtype
+      t.string     :status
+      t.string     :status_details
+      t.bigint     :tenant_id
```

Comment:
@syncrou don't we need t.timestamps here we would know when the field was created/updated by the user

Reply:
Yeah - will be force pushed here shortly.  It seems they were left out of the auto generated migration.